examples/jsm/utils/LDrawUtils.js

```diff
@@ -62,8 +62,11 @@ class LDrawUtils {
 					arr: [ geometry ]
 				};
 
+			} else {
+
+				geoms.arr.push( geometry );
+
 			}
-			else geoms.arr.push( geometry );
```

Comment:
This was causing a lint error because `else` was on a different line from the end brace which is why I changed it. I think an alternative would have been:

```js
} else geoms.arr.push( geometry );
```

Let me know if you have a preference

Reply:
I think it is ok as it is now.